ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/RequestConverter_Test.java

```diff
@@ -63,4 +63,54 @@ public void testGetRequestParameterListWithResponseType() {
         assertEquals(paramList.size(), 3);
         assertEquals(RequestHelper.paramListToString(paramList), "const ds3_client* client, const ds3_request* request, const ds3_get_service_response** response");
     }
+
+    @Test
+    public void testConvertDs3RequestWithRequestPayload() {
+        final Request testRequest = RequestConverter.toRequest(
+                new Ds3Request("CreateGetJobRequestHandler",
```

Comment:
Same as above, you can get this Ds3Request in the Ds3ModelFixtures.getRequestBulkGet()